code/modules/client/client_procs.dm

```diff
@@ -1230,6 +1230,16 @@
 		if("Set-Tab")
 			stat_tab = payload["tab"]
 			SSstatpanels.immediate_send_stat_data(src)
+		if("Listedturf-Scroll")
+			if(payload["min"] == payload["max"])
+				// Not properly loaded yet, send the default set.
+				SSstatpanels.refresh_client_obj_view(src)
+			else
+				SSstatpanels.refresh_client_obj_view(src, payload["min"], payload["max"])
+		if("Statpanel-Debug")
+			log_debug(payload)
```

Comment:
We probably don't want this on live. An href exploit could potentially use this to write anything to debug log, and therefore write with `rustg_log_write`, which to my knowledge, writes to the disk